### PR TITLE
Bug 1069007 - Label edit view toolbar buttons and make auto-enhance a toggle button.

### PR DESCRIPTION
**Pull request by eeejay**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9bea14d8a6e4)

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=635796d26d5b)

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9aa39e52a5d5)

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1719504efefb)

File: apps/gallery/index.html [hunk 1]
```diff
@@ -212,11 +212,11 @@ <h1 id="edit-title" data-l10n-id="edit"></h1>
...
-      <footer id="edit-toolbar">
-        <a id="edit-exposure-button" class="button" data-icon="brightness"></a>
-        <a id="edit-crop-button" class="button" data-icon="crop"></a>
-        <a id="edit-effect-button" class="button"></a>
-        <a id="edit-enhance-button" class="button"></a>
+      <footer id="edit-toolbar" role="toolbar">
+        <a id="edit-exposure-button" class="button" data-icon="brightness" data-l10n-id="exposure-button" role="button"></a>
+        <a id="edit-crop-button" class="button" data-icon="crop" data-l10n-id="crop-button" role="button"></a>
+        <a id="edit-effect-button" class="button" data-l10n-id="effect-button" role="button"></a>
+        <a id="edit-enhance-button" class="button"  data-l10n-id="enhance-button" role="button" aria-pressed="false"></a>
```
Comment (punamdahiya) [inline apps/gallery/index.html]: Similar comment as bug 1068984 patch, If simple consider changing links to button element.

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=23945cb25c74)

**Pull request merged**